src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -83,4 +83,16 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+   default String getPrintableString(Printable... printables){
+       String concatenatedString = "";
+       for (Printable printable : printables) {
+           concatenatedString += printable.getPrintableString();
+           concatenatedString += " ";
```

Comment:
Strings were supposed to be comma separated. Consider using String.join or StringJoiner